pkg/lib/verifier/rule_builder_util.go

```diff
@@ -178,33 +243,52 @@ func getTagCertificateTrustedRule(assetTagCACertificates *x509.CertPool, flavor
 
 	rule, err = rules.NewTagCertificateTrusted(assetTagCACertificates, &flavor.External.AssetTag.TagCertificate)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "Could not create the TagCertificateTrusted rule")
 	}
 
 	return rule, nil
 }
 
-func getPcrEventLogIncludesRules(pcrs []types.PcrIndex, flavor *hvs.Flavor, marker common.FlavorPart) ([]rules.Rule, error) {
+//getPcrEventLogIncludesRules method will create PcrEventLogIncludesRule and return the rule
+//return nil if error occurs
+func getPcrEventLogIncludesRules(pcrs []types.PcrIndex, flavor *hvs.Flavor, pcrLogData *types.PCRS, marker common.FlavorPart) ([]rules.Rule, error) {
 
 	var results []rules.Rule
```

Comment:
Using variable names as 'rules' instead of 'results' is showing error because we are using package name called 'rules'. Hence it is conflicting. Can we use some other name like 'pcrRules'?

Reply:
yes